src/components/info-modal/widgetInfo.js

```diff
@@ -1,55 +1,61 @@
 export const widgetInfo = {
+    'mangrove_extent': {
+      Title: '<strong>Global extent of mangroves for select years from 1996 to 2016.</strong>',
```

Comment:
`<strong>` for titles are wrong, it should be an `<h2>` or something like that. Anyway, we should remove it because it should be in the template.